tests/acceptance/blockchain/interfaces/test_transacting_power.py

```diff
@@ -75,9 +63,6 @@ def test_transacting_power_sign_transaction(testerchain):
                              signer=Web3Signer(testerchain.client),
                              account=eth_address)
 
-    assert power.is_active is False
-    assert power.is_unlocked is False
```

Comment:
How come we removed the assertions?

Reply:
In this PR I've removed locked/unlocked status tracking on the `TransactingPower` (leaving it up to the signer directly), and the notion of activation (it used to represent being cached on the chain interface.)